### PR TITLE
Adapt development documentation to new component behaviour

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

File: doc/development.md [hunk 1]
```diff
@@ -110,10 +110,8 @@ Before we can start you need to match some prerequirements on your host machine.
     3. Install all barclamps with this snippet
 
        ```bash
-       for d in $(find /root/barclamps -mindepth 1 -maxdepth 1 -type d)
-       do
-         CROWBAR_DIR=/opt/crowbar /opt/crowbar/bin/barclamp_install.rb --force ${d}
-       done
+       components=$(find /root/barclamps -mindepth 1 -maxdepth 1 -type d)
+       CROWBAR_DIR=/opt/crowbar /opt/crowbar/bin/barclamp_install.rb $components
```
Comment (MaximilianMeister) [inline doc/development.md]: cool. so there is no loop required here? barclamp_install can read arrays?

Comment (rsalevsky) [inline doc/development.md]: Yes, barclamp install can now handle multiple installations at once.

**Comment by MaximilianMeister** [comment]
havent tried it yet but I guess you did :) thanks for fixing this!!

:+1:

**Comment by tboerger** [comment]
LGTM

**Pull request merged**